pages/index.vue

```diff
@@ -1,26 +1,258 @@
 <template>
-  <section class="container">
-    <div>
-      <logo />
-      <h1 class="title">
-        akira
-      </h1>
-      <h2 class="subtitle">
-        The Linux Design Tool
-      </h2>
-      <div class="links">
-        <a href="https://nuxtjs.org/" target="_blank" class="button--green"
-          >Documentation</a
-        >
-        <a
-          href="https://github.com/nuxt/nuxt.js"
-          target="_blank"
-          class="button--grey"
-          >GitHub</a
-        >
+  <div class="wrapper">
+    <header>
+      <div>
+        <logo />
+        <nav>
+          <ul>
+            <li><a href="features">Features</a></li>
+            <li><a href="team">Team</a></li>
+            <li><a href="roadmap">Roadmap</a></li>
+            <li><a href="changelog">Changelog</a></li>
+            <li class="cta"><a href="donate">Donate</a></li>
+          </ul>
+
+          <ul class="social">
+            <li>
+              <a href="https://github.com/akiraux"
+                ><img
+                  src="~/assets/img/glyph-github.svg"
+                  alt="Link to Github"
+                />
```

Comment:
Same thing for this and all the others

Reply:
Upon further investigation I've found that Prettier has a flag --html-whitespace-sensitivity which I can set to ignore, 

https://prettier.io/blog/2018/11/07/1.15.0.html#whitespace-sensitive-formatting